config.go

```diff
@@ -45,11 +50,15 @@ type Config struct {
 
 // The ClientConfig describes a single Keywhiz client.  There are typically many of these per keysync instance.
 type ClientConfig struct {
-	Key     string `yaml:"key"`       // Mandatory: Path to PEM key to use
-	Cert    string `yaml:"cert"`      // Optional: PEM Certificate (If cert isn't in key file)
-	User    string `yaml:"user"`      // Optional: User and Group are defaults for files without metadata
-	DirName string `yaml:"directory"` // Optional: What directory under SecretsDir this client is in. Defaults to the client name.
-	Group   string `yaml:"group"`     // Optional: If unspecified, the global defaults are used.
+	Key        string `yaml:"key"`         // Mandatory: Path to PEM key to use
+	Cert       string `yaml:"cert"`        // Optional: PEM Certificate (If cert isn't in key file)
+	User       string `yaml:"user"`        // Optional: User and Group are defaults for files without metadata
+	DirName    string `yaml:"directory"`   // Optional: What directory under SecretsDir this client is in. Defaults to the client name.
+	Group      string `yaml:"group"`       // Optional: If unspecified, the global defaults are used.
+	MaxRetries uint16 `yaml:"max_retries"` // Optional: retry each HTTP call after non-200 response. Defaults to Keysync's config.
+	Timeout    string `yaml:"timeout"`     // Optional: timeout HTTP calls. Defaults to Keysync's config.
+	MinBackoff string `yaml:"min_backoff"` // Optional: wait time before first retry. Defaults to Keysync's config.
+	MaxBackoff string `yaml:"max_backoff"` // Optional: max wait time before retries. Defaults to Keysync's config.
```

Comment:
I don't think we need to be able to configure backoff per-client, and it's just extra complexity.  We don't allow configuring servers per-client, and it makes sense to configure retries and backoff with the server.